static/app/views/replays/detail/networkList.tsx

```diff
@@ -0,0 +1,80 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import {PanelTable} from 'sentry/components/panels';
+import Placeholder from 'sentry/components/placeholder';
+import {showPlayerTime} from 'sentry/components/replays/utils';
+import {t} from 'sentry/locale';
+import {EventTransaction} from 'sentry/types';
+
+type NetworkListProps = {
+  event: EventTransaction;
+  startTimestamp: number;
```

Comment:
Instead of asking for a whole `event`, can we just ask for the filtered network spans?

That should help to keep types strong.

Then instead of plucking just the timestamp value from event, pass in the real replay event

```
  <NetworkList
    event={replay.getEvent()}
    networkSpans={replay
      .getRawSpans()
      .filter(...)
      .map(...)}
  />
```


In the `.map()` that happens inside focusTab we can even import `type NetworkSpan` so the types are set right away. `NetworkSpan` does not need to be compatible with what `<Spans ../>` accepts. In the near future we're going to decide between keeping `<Spans>` or replacing it with `<NetworkList>`